styles/sections/_nav.scss

```diff
@@ -242,37 +245,38 @@
 
 // Social Navigation
 .social-nav-container {
-  display: flex;
-  flex-direction: column;
-  gap: 25px;
   margin-top: auto;
-  padding-bottom: 40px;
+  padding-bottom: 31px;
 
   @container body (min-width: 1024px) {
     margin-top: 0;
     padding-bottom: 0;
   }
 }
 
-.social-nav li:not(:first-child) a {
-  padding-left: 12px;
+.social-nav {
```

Comment:
Since the name of the container around the social media links was changed from `social-nav` to `social-nav-links`, can we remove these styles since this class no longer exists and are mostly repetitive of the `social-nav-links` styles?